### PR TITLE
Zoom-Pan-Hover controls + Getting to know the Modebar

### PR DESCRIPTION
**Pull request by MatthewHiscock**
As per @mariahhamel here is a new tutorial on the Modebar, plus some tweaks to the old one. With the staff shuffle, let me know if there's anybody else I should be tagging with this stuff from now on.
@chriddyp, @NicoleGrondin, @doeg

### CODE REVIEW

File: _posts/2015-09-01-getting-to-know-the-plotly-modebar.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+###Autoscale and reset axes
+
+When a graph is made using Plotly, we guess how the graph axes should be set. This is called “autoscale”. Sometimes our guess isn’t exactly what the graph creator wants. When a graph is made, the user can change the axes, and then save their graph. You can return to this saved form by clicking on Reset Axes.
+
+Clicking this zooms to include your Axes Range, if this has been set. If it has not been set it zooms to a setting that is optimized to include all the viewable data, the same as if Autoscale had been clicked.  | ![Reset Axes](/static/images/getting-to-know-the-plotly-modebar/reset-axes.png)
+
+Clicking Autoscale zooms the plot back to a setting that is optimized to include all the viewable data regardless of the Axes Range setting. | ![Autoscale](/static/images/getting-to-know-the-plotly-modebar/autoscale.png)
```
Comment (etpinard) [inline _posts/2015-09-01-getting-to-know-the-plotly-modebar.md]: very nice :+1:

**Comment by etpinard** [comment]
@MatthewHiscock looking good!

We should probably add a note saying that these are the *cartesian* buttons. 3D graphs, geographic maps and pie charts have different modebar buttons.

**Comment by MatthewHiscock** [comment]
@etpinard good point - that's a whole other kettle of fish. @mariahhamel - what do you figure about that? Another tutorial for those buttons, or add them to this?

**Comment by MatthewHiscock** [comment]
@chriddyp @etpinard - anything missing in the 3d section? It looks pretty skeletal right now.

File: _posts/2015-09-01-getting-to-know-the-plotly-modebar.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Clicking Compare data on hover will show you the data for all points with the same x-value. | ![Compare Data on Hover Button](/static/images/getting-to-know-the-plotly-modebar/compare-data-on-hover-button.png) | ![Compare Data on Hover](/static/images/getting-to-know-the-plotly-modebar/compare-data-on-hover.png)
+
+###3d charts
+
+Three-dimensional charts use a different set of buttons and adds several new ones. | ![3d Buttons](/static/images/getting-to-know-the-plotly-modebar/3d-buttons.png)
+
+Orbital Rotation rotates the plot around its middle point in three-dimensional space. | ![Orbital Rotation](/static/images/getting-to-know-the-plotly-modebar/orbital-rotation.png)
+  
+Turntable Rotation rotates the plot around its middle point while constraining one axis slightly. | ![Turntable Rotation](/static/images/getting-to-know-the-plotly-modebar/turntable-rotation.png)
```
Comment (etpinard) [inline _posts/2015-09-01-getting-to-know-the-plotly-modebar.md]: "while constraining the z axis" would be more precise.

**Comment by etpinard** [comment]
@MatthewHiscock 1 non- :no_entry_sign: comment  

:dancer:

File: _posts/2015-09-01-getting-to-know-the-plotly-modebar.md (hunk 2 above)
Comment (MatthewHiscock) [inline _posts/2015-09-01-getting-to-know-the-plotly-modebar.md]: @etpinard I would have written that but in practice, I can't seem to get that to be the case. It seems to "prefer" locking one of the axes, but it's not 100% constraining it. I've been using this one: https://plot.ly/10796/~chris/

**Comment by MatthewHiscock** [comment]
Adding in @bpostlethwaite

**Comment by bpostlethwaite** [comment]
Lets get it out there! Nice work, we can adjust as we go :)

**Comment by bpostlethwaite** [comment]
:dancer:

**Pull request merged**